src/script/oscar_scrapper/oscar_scrapper.py

```diff
@@ -92,6 +93,30 @@ def parse_movie_winner_nominees_special_category(
         raise ValueError(f"Category {category} is not a special category")
 
 
+"""
+  Get the oscar ceremony date from the oscars page
+
+  Args:
+    page_source: str: The page source of the oscars page
+"""
+
+
+def scrape_ceremony_date(
+    page_source: str,
+    
+): 
+    # Parse the page
+    soup = BeautifulSoup(page_source, "html.parser")
+
+    # Get the date of the ceremony (in a string format) 
+    ceremony_date_string = soup.find("div", class_="field--name-field-date-time").text.strip()
```

Comment:
Nice work :+1: 

I just believe that we can make the class name as a parameter of the function (with a default argument) to improve a little the modularity.

```suggestion
def scrape_ceremony_date(
    page_source: str,
    ceremony_date_class="field--name-field-date-time",
): 
    # Parse the page
    soup = BeautifulSoup(page_source, "html.parser")

    # Get the date of the ceremony (in a string format) 
    ceremony_date_string = soup.find("div", class_=ceremony_date_class).text.strip()
```